3-0-spring-mvc/3-0-1-hello-spring-mvc/src/main/java/com/bobocode/mvc/controller/NoteController.java

```diff
@@ -0,0 +1,30 @@
+package com.bobocode.mvc.controller;
+
+/**
+ * To make the controller work, you have to add some annotations:
+ * - @Controller - to let spring know that you want make a controller;
+ * - @RequestMapping("/notes") - to indicate the basic endpoint for all request methods;
+ * - @RequiredArgsConstructor (for using storage without explicit autowiring);
+ */
+public class NoteController {
```

Comment:
@erneman1 the same here, please inject `notes` and provide a JavaDoc instead of all comments. You can skip it if you want and I will add Java Docs later.